ftest.sh

```diff
@@ -373,6 +373,18 @@ if grep \"testsuite.setAttribute('name', 'avocado')\" \
 wq
 EOF
 fi
+# apply fix for https://github.com/avocado-framework/avocado/commit/bfab1c6b20f8489c343cfc8c8964d8905b0f72a1
```

Comment:
The commit message for that commit says it's Python 3 specific.  We are still using Python 2 for Avocado AFAIU.  Does this still apply to us?

Reply:
Ah missed that.  As one would expect, since it's python 3 related, this 'fix' did not make a difference. I'll be removing it.